AY2021S2-CS2103-T14-1/tp
# DG: Update Command History section

## What this does
Fixes #312 by adding an arrow to the relevant association label
Closes #189 by updating the command history storage subsection to use an activity diagram instead of a complex sequence diagram

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the Command History section of the DG makes sense and has correct diagrams.
